contrib/storage-cassandra/src/test/java/org/apache/drill/exec/store/cassandra/TestCassandraSuit.java

```diff
@@ -65,7 +66,10 @@ public static void tearDownCluster() {
 
   private static void startCassandra() {
     cassandra = new CassandraContainer<>("cassandra")
-        .withInitScript("queries.cql");
+      .withInitScript("queries.cql")
+        .withStartupTimeout(Duration.ofMinutes(2))
```

Comment:
Please keep same indent for the methods call chain. For example: 
```
    HiveParquetTableMetadataProvider metadataProvider = builder
        .withEntries(entries)
        .withHivePartitionHolder(hivePartitionHolder)
        .withHiveStoragePlugin(hiveStoragePlugin)
        .withReaderConfig(readerConfig)
        .withSchema(schema)
        .build();
```
https://github.com/apache/drill/blob/4aefcef2b665c5737471664912a26ef6ed9a6cfc/contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveDrillNativeParquetScan.java#L78